doc/classes/Geometry2D.xml

```diff
@@ -238,7 +238,7 @@
 			<param index="2" name="from_b" type="Vector2" />
 			<param index="3" name="to_b" type="Vector2" />
 			<description>
-				Checks if the two segments ([param from_a], [param to_a]) and ([param from_b], [param to_b]) intersect. If yes, return the point of intersection as [Vector2]. If no intersection takes place, returns [code]null[/code].
+				Checks if two line segments intersect, with line [code]a[/code] between [param from_a] and [param to_a] and line [code]b[/code] between [param from_b] and [param to_b]. If the line segments intersect, the point of intersection is returned as a [Vector2]. If no intersection takes place, [code]null[/code] is returned.
```

Comment:
Could remove `as a [Vector2]` as it's clear the point is already one of said type.

Reply:
I opted to leave it in because the function returns a Variant and it might be confusing not mention the expected return type anywhere.